__tests__/__resources__/properties/default_properties.yaml

```diff
@@ -95,6 +95,9 @@ zosjobs:
   # a physical sequential data set containing IEFBR14 JCL
   # that your user ID can submit
   iefbr14PSDataSet: zosjobs-iefbr14PSDataSet
+  # a path to a uss file containing IEFBR14 JCL
+  # that your user ID can submit
+  iefbr14USSFile: zosjobs-iefbr14USSFile
```

Comment:
Curious what you think about the idea of updating our test scripts (it may not make much sense to update the `cli-test-utils` package) to "prompt" for these new values before we run system tests?

I guess doing a `diff custom_properties.yaml default_properties.yaml` as a `pretest:system` will be sufficient for now.

The thing is that I always forget to update my custom_properties file, and I only realize I'm missing something when a test complains about `xyz` not being defined 😋

Reply:
That might be a nice enhancement, I just don't know the scope of the effort that would be required to prompt for anything that is missing, and that might be a little bit of a problematic behavior for something that is running in a CI environment.

Reply:
I agree with the CI part and not prompting (though not implemented for system tests yet) 😋 
Maybe just printing out the diff is sufficient 😋 
No need to do it in this PR though, we can address it later 😁